docs/docs/guides/reverse_proxy.md

```diff
@@ -0,0 +1,66 @@
+---
+id: reverse_proxy
+title: Setting up a Reverse Proxy
+---
+This guide outlines the configuration steps needed to expose your Frigate UI to the internet in a secure manner.
```

Comment:
We should definitely have a warning here that reverse proxies can be secure but security relies on the user setting it up correctly, the guide is an example and not meant to be thought of as a complete and secure setup, etc.

Reply:
Good spot. Added a caveat section and also removed language that inferred this was all of the answer.